### DISCUSSION TITLE
Is it possible to skip Fastify and just pass query and variables to Mercurius?

### DISCUSSION THREAD

**Discussion by thelinuxlich** [mercurius-js/mercurius#647] [Q&A]
I'm using a library which autogenerates a nice API on top of a GraphQL schema, [GenQL](https://genql.vercel.app/). I want to let the user create custom resolvers where they interact with the same API on the server-side, but skipping the HTTP layer, is it possible?

**Comment by mcollina** [comment]
I have used `app.inject()` in the past. It works great for this purpose. https://www.fastify.io/docs/latest/Testing/#benefits-of-using-fastifyinject
